packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -312,11 +312,6 @@ private static Request setHeadersForUrl(Uri url, UpdatesConfiguration configurat
             .header("Expo-JSON-Error", "true")
             .header("Expo-Accept-Signature", String.valueOf(configuration.expectsSignedManifest()));
 
-    // legacy manifest loads should ignore cache-control headers from the server and always load remotely
-    if (configuration.usesLegacyManifest()) {
-      requestBuilder = requestBuilder.cacheControl(CacheControl.FORCE_NETWORK);
```

Comment:
reminder that this PR should go under Breaking Changes in the changelog due to this change